docs/mddocs/README.zh-CN.md

```diff
@@ -4,7 +4,7 @@
 </p>
```

Comment:
change all links to "llama_cpp_quickstart.zh-CN.md" to  "llamacpp_portable_zip_gpu_quickstart.md" in this file